stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -65,6 +68,8 @@
   private ChildNodeRemovedEvent mCachedChildNodeRemovedEvent;
   private ChildNodeInsertedEvent mCachedChildNodeInsertedEvent;
 
+  private final Map<String, List<Integer>> mSearchResults;
```

Comment:
I think we're fine to use `List<Integer>` in this case, although it's a little circumstantial. We're pulling the `Integer`s from `mObjectIdMapper.getIdForObject()` which is already storing them as `Integer` in its mObjectToIdMap ... in other words, we're not adding any extra overhead.